OasisDEX/oasis-borrow
# Aave stop loss open flow

# [Aave stop loss open flow](https://app.shortcut.com/oazo-apps/story/7706/aave-sl-in-open-position-flow)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added handling of stop loss step in aave open multiply position flow
  
## How to test 🧪
  <Please explain how to test your changes>

- try to open aave multiply position for different token pairs
- extra (skippable) step with stop loss should be present
- when SL level is selected, order information section should contain both information about position and stop loss
- when SL level is selected two transactions should be triggered, one after another (the same way as for maker)
